Updated mermaid balloon styling

mateuszzagorski opened the pull request:
Restored toolbar height to the previous value, spacing around info icon updated. Closes: #307 & #309.

mlewand commented on src/app/theme/panel.css:
```diff
@@ -35,11 +35,11 @@
 
 .ck.ck-balloon-panel [aria-label="Mermaid toolbar"] .ck-button {
-	line-height: 1;
+	line-height: 21px;
```
If I'm not mistaken this is exactly the same value that is assigned in generic styles for `.ck-button`.

If that's the case there's no reason to explicitly override it for the same value - just remove the rule. Less is better.

mlewand commented on src/app/theme/panel.css:
```diff
@@ -35,11 +35,11 @@
 
 .ck.ck-balloon-panel [aria-label="Mermaid toolbar"] .ck.ck-toolbar__separator {
-	margin-right: var(--ck-spacing-medium);
+	margin-right: var(--ck-spacing-small);
```
I think that with left/right margin set to 0 it looks better:

![](https://user-images.githubusercontent.com/5353898/157664168-1d0a7904-919a-4950-895d-ce019bfe7407.png)

vs

![image](https://user-images.githubusercontent.com/5353898/157664274-7f58e01f-9486-494d-ae7b-5fd487c9353d.png)

mlewand reviewed (changes requested):
Icons are now unevenly aligned along the vertical axis.

![image](https://user-images.githubusercontent.com/5353898/157663289-1b39fc7b-d536-43ea-a4b8-e202789f91f6.png)

There's a lot of extra space above.

mlewand reviewed (commented):
I also noticed that this is a result of a wider problem:

![image](https://user-images.githubusercontent.com/5353898/158136678-d72d978c-ba07-4099-aa99-0a4639c8aee2.png)

So any balloon toolbar will render poorly (note space on the right, and uneven vertical alignment). I made an issue to track this problem too https://github.com/ckeditor/github-writer/issues/309

My proposal is to turn this PR into a fix of a general problem (#309).

I'm not sure if we should fix it with margin tweaks. I'd try first to see if we can retain the original display inline-flex - which works really well in vanilla CKE5.

mateuszzagorski commented:
Updated various CSS styles to improve mermaid balloon styling as well as toolbars in general.  
One of the changes would be the use of a separator styling - which we introduced for the mermaid toolbar, in the main toolbar as well - you can see it below:

![](https://s3.us-west-2.amazonaws.com/secure.notion-static.com/c355932f-005f-401d-b621-2187f173d90c/Untitled.png?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Content-Sha256=UNSIGNED-PAYLOAD&X-Amz-Credential=AKIAT73L2G45EIPT3X45%2F20220316%2Fus-west-2%2Fs3%2Faws4_request&X-Amz-Date=20220316T095101Z&X-Amz-Expires=86400&X-Amz-Signature=b894646e230e282dfac04b4d8bd1815c0b939d7b941a2fbbea57fb7f742b1fd2&X-Amz-SignedHeaders=host&response-content-disposition=filename%20%3D%22Untitled.png%22&x-id=GetObject)

mlewand reviewed (commented):
The change indeed fixed the styling for the mermaid toolbar but only of it.

## Mermaid

old:
![image](https://user-images.githubusercontent.com/5353898/170987346-0c91c80d-efe7-4eaa-a32a-6dbf076ff41d.png)

new:
![image](https://user-images.githubusercontent.com/5353898/170987292-12128004-1eed-442a-a7b1-f91842f04c3e.png)

## Table toolbar

old:
![image](https://user-images.githubusercontent.com/5353898/170987456-ee89f8d2-14ef-4702-ba5c-9d48ae4396d6.png)

new:
![image](https://user-images.githubusercontent.com/5353898/170987461-e7b4d448-c292-43eb-bd43-a883564c4222.png)

There's no change, so #309 remains unfixed by this PR.

---

What I see is source for this extra right margin is this thing:

![image](https://user-images.githubusercontent.com/5353898/170994313-783e3ba9-966b-45d0-a7b0-770c7a32776e.png)

Why is it there? It looks like a source of the problem. Shouldn't we change this?

I recall that it might be meant for the main toolbar, like here:

![image](https://user-images.githubusercontent.com/5353898/170994619-29b8c66e-7d45-4426-b18f-6220eeb0d62e.png)

In which way the proper solution is to add this margin-right **only** for this "main" toolbar - make a selector that will match only it.
